bigquery/snippets/snippet.go

```diff
@@ -142,10 +141,86 @@ func createTable(client *bigquery.Client, datasetID, tableID string) error {
 	if err := table.Create(ctx, &bigquery.TableMetadata{Schema: schema}); err != nil {
 		return err
 	}
+	return nil
+}
+
+func createTableExplicitSchema(client *bigquery.Client, datasetID, tableID string) error {
+	ctx := context.Background()
+	// [START bigquery_create_table]
+	// Represent email_addrs as an array of strings, rather than a single address
+	sampleSchema := bigquery.Schema{
+		{Name: "full_name", Type: bigquery.StringFieldType},
+		{Name: "age", Type: bigquery.IntegerFieldType},
+	}
+
+	metaData := &bigquery.TableMetadata{
+		Schema:         sampleSchema,
+		ExpirationTime: time.Now().AddDate(1, 0, 0), // Table will be automatically deleted in 1 year
+	}
+	tableRef := client.Dataset(datasetID).Table(tableID)
+	if err := tableRef.Create(ctx, metaData); err != nil {
+		return err
+	}
 	// [END bigquery_create_table]
 	return nil
 }
 
+func createTableEmptySchema(client *bigquery.Client, datasetID, tableID string) error {
+	ctx := context.Background()
+	// [START bigquery_create_table_without_schema]
+	// Create the table as a partitioned table
+	metaData := &bigquery.TableMetadata{
+		TimePartitioning: &bigquery.TimePartitioning{
```

Comment:
Are planning on using this sample at https://cloud.google.com/bigquery/docs/creating-partitioned-tables too? We should have another region tag for that.

Note: it is possible to wrap a sample in multiple region tags and devsite will strip the inner ones out from the outer include.

```
// [START bigquery_create_table_partitioned_by_ingestion_time]
// [START bigquery_create_table_without_schema]
... sample code ...
// [END bigquery_create_table_without_schema]
// [END bigquery_create_table_partitioned_by_ingestion_time]
```

@alixhami I think we need some new region tags in samples tracker for the samples at https://cloud.google.com/bigquery/docs/creating-partitioned-tables

Reply:
`bigquery_create_table_without_schema` should be the bare minimum code to create a table in order to be consistent with the [canonical sample](https://cloud.google.com/bigquery/docs/tables#table-no-schema), meaning that it should not include partitioning.

Another thing to note is that really long region tags tend to be cut off in the sample tracker tables right now, so I prefer when they're as short as possible. Is there any shorter way that still makes sense? Ex. `bigquery_table_partition_ingestion_time`

Reply:
To unblock alix's related changes elsewhere to docs, I'm going to simplify this and think about a better way to approach the combination problem with the samples.

Reply:
Thanks Seth. Re: shorter table names. I like to have a verb just because there are so many operations one might want to do with a table. I'd lean towards `bigquery_create_table_partition_injest` for this particular one.